exec-service-wasmer/src/wasmer_breakpoints.rs

```diff
@@ -15,71 +15,76 @@ pub(crate) const BREAKPOINT_VALUE_NO_BREAKPOINT: u64 = 0;
 #[allow(dead_code)]
 pub(crate) const BREAKPOINT_VALUE_EXECUTION_FAILED: u64 = 1;
 pub(crate) const BREAKPOINT_VALUE_OUT_OF_GAS: u64 = 4;
-#[allow(dead_code)]
 pub(crate) const BREAKPOINT_VALUE_MEMORY_LIMIT: u64 = 5;
 
 #[derive(Clone, Debug, MemoryUsage)]
-struct BreakpointsGlobalIndex(GlobalIndex);
-
-impl BreakpointsGlobalIndex {
-    fn breakpoints_global_index(&self) -> GlobalIndex {
-        self.0
-    }
+struct BreakpointsGlobalIndexes {
```

Comment:
There is only one global index managed by this middleware, consider renaming to `BreakpointsGlobalIndex`.

Reply:
done